src/tests/ftest/util/ior_utils.py

```diff
@@ -111,6 +110,9 @@ def __init__(self):
         self.daos_chunk = FormattedParameter("--daos.chunk_size {}", 1048576)
         self.daos_oclass = FormattedParameter("--daos.oclass {}")
 
+        # Setting object class for MPIIO as SX by default
```

Comment:
Is this needed?  Can the value in self.daos_oclass be used to set the `IOR_HINT__MPI__romio_daos_obj_class` env?  Is there ever a need for the env and the ior command line option to specify different object classes?

If it is not needed, should `self.daos_oclass` be defined with a default of `SX` or should the env not be set if `self.daos_oclass..value == None`